BUILD

```diff
@@ -0,0 +1,25 @@
+# This is additionally exposed for the test rule in trlc.bzl
+exports_files(
+    ["trlc.py"],
+    visibility = ["//visibility:public"],
+)
+
+py_binary(
+    name = "trlc_binary",
+    srcs = [
+        "trlc.py",
+    ],
+    args = ["--use-cvc5-binary $(location @cvc5)"],
+    data = ["@cvc5"],
+    main = "trlc.py",
+    visibility = ["//visibility:public"],
+    deps = [
+        "//trlc",
+    ],
+)
+
+alias(
+    name = "cvc5",
+    actual = "@cvc5",
+    visibility = ["//visibility:public"],
+)
```

Comment:
Why is the alias necessary?

Reply:
Using bzlmod, dependencies of a module a not visible in a dependent module.

Meaning, if I have another module that utilizes trlc, I could not see and use `@cvc5` because it is a dependency of TRLC. Since we use this in a macro, and the macro is executed in the context of `another module`, you need this alias. This way you can reference a TRLC target in the macro, which is known, and this alias is then evaluated in the context of TRLC which knows `@cvc5` because its a direct dependency of it.